src/components/timeline/TimelineContextMenu.svelte

```diff
@@ -63,13 +72,27 @@
     const newVerticalGuide = createVerticalGuide(timelines, cursorDOY);
     dispatch('updateVerticalGuides', [...verticalGuides, newVerticalGuide]);
   }
+
+  function updateSimulationStartTime(date: Date) {
+    const doyString = getDoyTime(date, false);
+    const newSimulation: Simulation = { ...simulation, simulation_start_time: doyString };
+    effects.updateSimulation(newSimulation);
+  }
+
+  function updateSimulationEndTime(date: Date) {
+    const doyString = getDoyTime(date, false);
+    const newSimulation: Simulation = { ...simulation, simulation_end_time: doyString };
+    effects.updateSimulation(newSimulation);
```

Comment:
Not in this PR, but should we allow sim end time to be set before sim start time? Seems to throw an error so we should probably protect against that.

Reply:
Agreed, I can make a new issue for that.

Reply:
https://github.com/NASA-AMMOS/aerie-ui/issues/563